docs/_may_milestone_.md

```diff
@@ -0,0 +1,62 @@
+## Monthly Milestone - May 2020
+
+### Overview
+
+### Goals
+
+- [ ] Migration Guide
+
+  - [ ] How to migrate LB3 components #3955 _depends on the spike result in
+        #5004_
+  - [ ] How to run LB3 app tests when the app is mounted in a LB4 project #3978
+        _depends on the spike reuslt in #4099_
+
+- [ ] Feature Parity
+
+  - [ ] Model.toObject() should preserve prototypes (e.g. Date and ObjectID
+        values) #3607
+
+- [ ] Bugs
+
+  - [ ] defining a nested array model property using @property.array decorator
+        crashes the app #4754
+
+- [ ] Docs
+
+  - [ ] [Docs] Add debug string information #4845
+  - [ ] [Docs] Add "error handling" page #4846
+
+- [ ] Miscellaneous
+
+  - [ ] How to build models, repositories and controllers dynamically at runtime
+        #4296
+  - [ ] Spike: robust handling of ObjectID type for MongoDB #3456
+  - [ ] Integration with TypeORM #4853
+
+- [ ] PRs to review
+  - [ ] feat(repository): hasManyThrough #4438
+  - [ ] [RFC] Enable parallel mocha testing #5011
+  - [ ] feat(http-server): adds http/2 support #4989
```

Comment:
@raymondfeng can you please add 2-3 of your pull requests you would like the team to focus on reviewing during May?

Reply:
Would also like to get this refresh token PR reviewed: https://github.com/strongloop/loopback4-example-shopping/pull/537